uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -376,9 +383,10 @@ Arguments:
 sub isDicom {
     my $this         = shift;
     my ($dicom_file) = @_;
-    my $file_type    = $this->runCommand("file $dicom_file");
+    my $cmd    = "file $dicom_file";
+    my $file_type    = `$cmd`;
     if ( !( $file_type =~ /DICOM/ ) ) {
-        print "not of type DICOM" if $this->{'verbose'};
+        print "\n Not of type DICOM \n";
```

Comment:
@mohadesz,

We print *error* messages irrespective of what $verbose is set to.